corehq/apps/userreports/tasks.py

```diff
@@ -479,13 +474,15 @@ def _mark_config_to_remove(config_id, indicator_ids):
                 )
                 indicator.save()
 
-        datadog_counter('commcare.async_indicator.processed_success', len(processed_indicators))
-        datadog_counter('commcare.async_indicator.processed_fail', len(failed_indicators))
-        datadog_histogram(
```

Comment:
histogram here converted to a total + count to allow calculating an average and 'cumulative rate'